packages/components/Badge/src/__tests__/__snapshots__/Badge.test.tsx.snap

```diff
@@ -352,8 +352,8 @@ exports[`CounterBadge component tests CounterBadge shows zero 1`] = `
     Object {
       "alignItems": "center",
       "alignSelf": "flex-start",
-      "backgroundColor": "#0078d4",
-      "borderColor": "transparent",
+      "backgroundColor": "#185abd",
```

Comment:
Hmm do you know why the background is changing?

Reply:
Okay so this was a result of the change moving some of the common global tokens into one shared file:

State before the change:
- global.windows.json: brand.primary = #0078d4
- global.win32.json: brand.primary = #185abd
- global.macos.json: brand.primary = #0078d4
- (android/ios don't have brand.primary)

State after the change:
- global.brand.legacy.json: brand primary color = #185abd (matches the old global.win32.json value, but not windows/macOS)

Since all our tests use default theme/windows tokens this changed all the usages of brand.primary - there should be no changes on the win32 side though